### PR TITLE
Els e2e orders [delivers #158074616]

### PR DESCRIPTION
**Pull request by tinyels**
## Description

happy path tests for orders in service member flow

## Reviewer Notes

I also made some improvements to how signing in as a new user works and refactored to make it easier to run the tests I am interested in.

file uploads are not easy to do e2e tests. Since we didn't write the component we are using I am not that concerned for this case, but we will need some way to get the uploads in place to do the next story
## Code Review Verification Steps

* [x ] All tests pass.

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Cypress.Commands.add('signInAsUser', userId => {
+  cy.visit('/');
+  cy
+    .request({
+      method: 'POST',
+      url: '/devlocal-auth/login',
+      form: true,
+      body: { id: userId },
+    })
+    .then(() => cy.visit('/'));
```
Comment (chrisgilmerproj) [inline cypress/support/commands.js]: for these sign ins, is it more important to use the api or to use the buttons? More clearly, are we hoping to test the API directly or the React/Redux code?

Comment (tinyels) [reply cypress/support/commands.js]: cypress recommends not using the buttons. Also, the code for the buttons here only exists in a dev environment, so it is not a very valuable thing to test

**Comment by tinyels** [comment]
These tests are pretty minimal--we should have a conversation about what sorts of things we want to verify in e2e tests.

File: cypress/support/commands.js (hunk 1 above)
Comment (chrisgilmerproj) [reply cypress/support/commands.js]: That's actually really valuable to know when writing tests.  Thanks!

**Pull request merged**